CI failed

```
AssertionError: ["In file '''/home/runner/work/rosdistro/rosdistro/foxy/distribution.yaml''': Could not validate source entry for repo py_trees_ros_tutorials with error [[[Could not validate repository with url https://github.com/splintered-reality/py_trees_ros_tutorials.git and version release/2/1.x from entry at line 4118. Error reason: No branch found matching release/2/1.x]]]"] is not false
```

--- a/foxy/distribution.yaml
+++ b/foxy/distribution.yaml
@@ -4119,7 +4119,7 @@ repositories:
       test_pull_requests: true
       type: git
       url: https://github.com/splintered-reality/py_trees_ros_tutorials.git
-      version: release/2/1.x
+      version: release/2.1.x
     status: maintained
   py_trees_ros_viewer:
     doc:

